src/app.js

```diff
@@ -156,6 +155,8 @@ module.exports.run = function (worker) {
       return next();
     }
 
+    // I'm not sure this code can actually be reached, as long as isAuthorized is used after getContextMiddleware,
+    // since an exception is thrown on missing tokens or if getContext(..) fails.
```

Comment:
> I agree, - to me it also seems like this will not be reached.

Everything related to isAuthorized can probably be removed. Not 100% sure, so it's probably best to leave it to after 1.0, and then maybe somebody could clean up the mess app.js has turned into. :-)